tests/api/data/__init__.py

```diff
@@ -0,0 +1,15 @@
+import json
```

Comment:
I wanted to ask you about this :)

I noticed that you don't separate clearly the tests by function under test. I find it essential to have maintainable tests.
Another option would be to organize tests in classes with the name of the function under test.
I prefer the way I have done because it allows me also to have a scoped (e.g. `api`) data module, and I find it easier to navigate through files than scrolling long files.

But you are the code maintainer, so the choice is yours, I just wanted to show you an alternative :)

If you do like it, I can re-structure all tests in `tests/test_api.py` accordingly. Otherwise I will make the tests I wrote fit in the pre-existing structure.

Reply:
TBH, I would refrain from refactoring all the tests into anything :) If you mean tests where I test couple of related commands (add+remove, register+deprecate), I do that because those are connected. I could write fixtures OFC, but then the number of repetitions would be greater I think (like when you need to write a test + a fixture that does roughly the same). You can think of it like preferring integration tests to unit tests for those cases, if that's analogy is correct here :)

Reply:
If you wish to do it, we can discuss it OFC. But in this case I need more details to better understand WDYM. Do you mean creating a separate files for each command? Like `test_show.py`, `test_annotate.py`, etc?

Reply:
You could say that. For each public function in a module that you want to test.

What is fixtures OFC?

Reply:
LOL sorry
OFC = of course

Reply:
LOL ok :)

Just let me know what you prefer about the test structure. Perhaps one more idea could be to split inside the folder `tests` into `tests/integration_tests` (the pre-existent) and `tests/unit_tests` (the one I wrote).

I am good with anything, even deleting all my tests :)

Reply:
Ok, I see the value in splitting tests in multiple files like `test_show.py`, `test_annotate.py`, but since this will not work not for all files (e.g. `test_showcase.py` is clearly an integration test, `test_cli.py` have huge `test_commands` function that makes things faster repeating different read-only commands on a single repo, etc), I'm not sure we should do that right away. Since the project is quite small and there are not so many tests, let's stick to the current structure for now. If you want to re-write tests to fit to what you suggest, let's do it in another PR later.

Also, please rename `data` into `resources` to be consistent [with MLEM ](https://github.com/iterative/mlem/tree/main/tests).

Reply:
done